ksim_kbot/common.py

```diff
@@ -200,6 +200,64 @@ def observe(self, state: ksim.ObservationState, rng: PRNGKeyArray) -> Array:
         return feet_contact_12.flatten()
 
 
+@attrs.define(frozen=True)
+class ActPosObservation(ksim.Observation):
+    """Observation for a specific joint's action and position."""
```

Comment:
update this comment to say something about how you expect this to be used - i.e., for decluttering the observation space